package/raphael-fusioncharts.js

```diff
@@ -5704,48 +6087,76 @@ if (typeof _window === 'undefined' && typeof window === 'object') {
                         from[attr] = element.attr(attr);
                         (from[attr] == null) && (from[attr] = availableAttrs[attr]);
                         to[attr] = params[attr];
-                        change = false;
+                        change = true;
```

Comment:
@pratyushfc  Do not set 'change' to true forcefully. It will effect performance.